Makefile

```diff
@@ -11,6 +11,11 @@ generate: sync
 	./hack/bootstrap-ginkgo.sh
 	(cd tools/openapispec/ && go build)
 	tools/openapispec/openapispec --dump-api-spec-path api/openapi-spec/swagger.json
+	docker run -u `stat -c "%u" hack/gen-swagger-doc/` --rm \
```

Comment:
Since it is not required to check in the html documentation of  the specification, I would prefer to have an extra makefile target. Something like: 

```
apidocs: generate
    docker run ...
```

Reply:
where should I call this target?
as part of travis scripts? or as travis deploy script ?

Reply:
As a normal build step is fine for me.

Reply:
done